src/components/Tooltip/Tooltip.jsx

```diff
@@ -136,14 +145,20 @@ export default class Tooltip extends React.PureComponent {
       content,
       getContainer: getContainer || this.getContainer,
       moveBy,
-      tooltipClassName: `monday-style-arrow monday-style-arrow-${theme} padding-size-${paddingSize}`,
+      tooltipClassName: cx(
+        styles[camelCase("monday-style-arrow")],
```

Comment:
there is no easier way to perform these conversions (without convert already known constant to camel case every time?

Reply:
Script does that - that's the example of human error :) Script won't replace this prop cause it's custom prop.
Actually we don't even need this at all - such class doesn't exist in the stylesheet and serves only for overrides, so I removed this line `styles[camelCase("monday-style-arrow")]`